rio_tiler/io/base.py

```diff
@@ -259,6 +260,8 @@ class MultiBaseReader(SpatialMixin, metaclass=abc.ABCMeta):
 
     assets: Sequence[str] = attr.ib(init=False)
 
+    ctx: Any = attr.ib(init=False, default=contextlib.nullcontext)
```

Comment:
in practice `ctx` will be `rasterio.Env` but this is not mandatory and could be let to `contextlib.nullcontext` is the MultiBaseReader class do not use a rasterio based reader